content/PHPDataObjects/DataObjects.facts.md

```diff
@@ -0,0 +1,40 @@
+### Facts for PHP Data Objects lesson:
+
+`PHP Data Objects (PDO)` defines an interface in PHP to access certain databases.
+
+`PDO::query()` executes an SQL statement in a single function call and returns a result set as a `PDOStatement` object or `false` on failure.
+
+The `PDOStatement` class represents a prepared statement and returns an associated result set after the statement is executed.
+
+The example code below shows how to execute queries in PHP using `PDO::query()`.
+
+```php
+// student.php
+<?php
+    require_once("connection.php");
+
+    $sql = "CREATE TABLE IF NOT EXISTS students (PRIMARY KEY (student_id), student_id INT GENERATED ALWAYS AS IDENTITY, first_name VARCHAR(80), last_name VARCHAR(80), birth_date DATE, gender VARCHAR(10))";
+    
+    try {
+        $stmt = $conn->query($sql);
```

Comment:
where is `$conn` coming from? This will trigger an error if we run this code.

Reply:
@cerilacanda Since this is the 2nd lesson, and the first lesson isn't using the `Connection` class, I think we need to include the `Connection.php` code in here so the learners can know what was on that file.

Reply:
This is noted. Thanks.